### DISCUSSION TITLE
when edgedb v3.10 will publish?

### DISCUSSION THREAD

**Discussion by AnsonHwang86** [geldata/gel#5021] [General]
when edgedb v3.10 will publish? I am waiting for several bugs fixed in that version, especially custom function over http doesn't work

**Comment by raddevon** [comment]
The only ETA I have at the moment for any 3.x is the beta which should be live this month. We'll give an ETA for the full release of 3.0 as we get a bit closer.

**Comment by elprans** [comment]
I think the OP meant 2.10.  We plan a release this week.

**Reply by AnsonHwang86** [reply]
Thanks, Yes I mean 2.10, sorry for typo.
